modules/apps/knowledge-base/knowledge-base-service/src/main/java/com/liferay/knowledge/base/internal/exportimport/data/handler/KBArticleStagedModelDataHandler.java

```diff
@@ -102,6 +102,14 @@ public String getDisplayName(KBArticle kbArticle) {
 		return kbArticle.getTitle();
 	}
 
+	@Override
+	public int[] getExportableStatuses() {
```

Comment:
Hi @adolfopa 

I tried this but it fail me the second time I was trying to publish the site to live.

Can you check it if it fails to you too?

Reply:
I also got an error when editing and republishing it.

![image](https://github.com/liferay-lima/liferay-portal/assets/81972512/39580c68-e4f0-40e8-a7da-cba8623e4273)